src/renderPreview/renderPreviewImpl.ts

```diff
@@ -11,6 +11,18 @@ const injectMathCss = (contentWindow: Window) =>
     contentWindow.document.head.appendChild( createLinkEl(href) )
   )
 
+// add link elements for default-light.min.css and default-dark.min.css
+// and add appropriate attributes to select the correct one 
+// based on the system's color scheme
+const injectHighlightCss = (contentWindow: Window) => {
+  const lightLink = createLinkEl('./highlight.js/default-light.min.css')
+  lightLink.setAttribute('media', '(prefers-color-scheme: light)')
+  contentWindow.document.head.appendChild(lightLink)
+  const darkLink = createLinkEl('./highlight.js/default-dark.min.css')
+  darkLink.setAttribute('media', '(prefers-color-scheme: dark)')
+  contentWindow.document.head.appendChild(darkLink)
+}
+
```

Comment:
Similar to `injectMathCss` except for the syntax highlighting themes we also need to set them up to be selected based on the system light/dark mode.